pakyow-support/lib/pakyow/support/defineable.rb

```diff
@@ -61,6 +82,12 @@ def freeze
         super
       end
 
+      # Provide default make method
+      #
+      def make(*args, &block)
```

Comment:
Ok sorry. Now that I take a closer look at `stateful`, I realize my mistake. The `object` referenced there doesn't need to be a `Defineable` object, so these defaults don't make sense.

So, I guess we just need to document that a stateful object has to respond to `make`, that `make` should return an instance of that type, and that it should `instance_eval` the block passed.